packages/subcommands/src/register.ts

```diff
@@ -0,0 +1,44 @@
+import { Plugin, postInitialization, postLogin, SapphireClient, Store } from '@sapphire/framework';
+import { SubcommandsPluginOptions, SubcommandsHandler } from './lib/utils/SubcommandsHandler';
+import { join } from 'path';
+import type { ClientOptions } from 'discord.js';
+
+export class Subber extends Plugin {
+	public static [postInitialization](this: SapphireClient, options: ClientOptions): void {
+		Store.injectedContext.subcommandsHandler = new SubcommandsHandler(this, options.subcommands ?? {});
+		this.events.registerPath(join(__dirname, 'events'));
```

Comment:
`/register` assumes you have `@sapphire/framework` installed, which by itself, defines the event store.

Reply:
Left at it is.

Reply:
@kyranet It does indeed, however, users may have their own event store, under a different name or otherwise. I do ultimately think it would be better to leave to the user.